### ISSUE TITLE
[testability] Deleting record set from inside set does not immediately close it

### ISSUE THREAD

**Issue by chanulee1** [specify/specify7#2519]
https://user-images.githubusercontent.com/103299204/204968470-5c7189f8-5fe0-4bce-ad31-5d7f8da0a91a.mp4

https://kufishnotests-testability.test.specifysystems.org/specify/overlay/record-sets/

**Comment by chanulee1** [comment]
Fixed. However, deleting last record in RS also navigates back to home page. Prior to fix, deleting the last record would just open up the "Creating new record" form in the RS.


https://user-images.githubusercontent.com/103299204/205243572-55c34967-7c34-40af-9e4c-1952c3337280.mp4



https://kufishnotests-testability.test.specifysystems.org/specify/view/collectionobject/61177/?recordsetid=113

**Comment by maxxxxxdlp** [comment]
@grantfitzsimmons Which of the two behaviors is desired for when you delete last record set item?

**Comment by grantfitzsimmons** [comment]
I don't think this will be a very common workflow, but I think the most logical thing would be to delete the record set if all items within are removed.

I think the new behavior is good as long as it deletes the record set as well.

**Comment by maxxxxxdlp** [comment]
It does not remove the record set. Will open a bug for this for future release
